products/memgraph/v1.2.0/how_to_guides/use-query-modules-provided-by-memgraph.md

```diff
@@ -1,81 +1,69 @@
-## How to Use Query Modules Provided by Memgraph?
+## Use Query Modules Provided by Memgraph?
 
-Memgraph supports extending the query language with user-written procedures.
-These procedures are grouped into modules, which can be loaded either on startup
-or afterwards using the built-in utility module.
-
-### Utility Query Module
-
-Query procedures that allow the users to gain more insight into other query
-modules and their procedures are written under our utility `mg` query module.
-This module offers three procedures with the following signatures:
-
-* `mg.procedures() :: (name :: STRING, signature :: STRING)`: lists loaded
-  procedures and their signatures
-* `mg.load(module_name :: STRING) :: ()`: loads or reloads the given module
-* `mg.load_all() :: ()`: loads or reloads all modules
-
-For example, invoking `mg.procedures()` from openCypher like so:
+Memgraph Community edition comes with a set of Python query modules based on
+the [NetworkX](https://networkx.github.io/) library of algorithms. The modules
+are already packaged within all Memgraph packages, but `NetworkX` might have to
+be installed by running the following command:
 
-```opencypher
-CALL mg.procedures() YIELD *;
 ```
-
-might yield the following result:
-
-```plaintext
-+---------------------+-------------------------------------------------------------------+
-| name                | signature                                                         |
-+---------------------+-------------------------------------------------------------------+
-| louvain.communities | louvain.communities() :: (community :: INTEGER, id :: INTEGER)    |
-| louvain.modularity  | louvain.modularity() :: (modularity :: FLOAT)                     |
-| mg.procedures       | mg.procedures() :: (name :: STRING, signature :: STRING)          |
-| mg.load             | mg.load(module_name :: STRING) :: ()                              |
-| mg.load_all         | mg.load_all() :: ()                                               |
-+---------------------+-------------------------------------------------------------------+
+pip3 install networkx
```

Comment:
I would add a bit more details because `pip3 install networkx` might not work in all cases because Memgraph is looking for a system python3... I would add a bit more details about all that.